sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/ModuleIdentityLifecycleSamples.cs

```diff
@@ -109,7 +108,7 @@ public async Task<ModuleIdentity> CreateModuleIdentityAsync(DeviceIdentity devic
                 // Call APIs to create the module identity.
                 Response<ModuleIdentity> response = await IoTHubServiceClient.Modules.CreateOrUpdateIdentityAsync(moduleIdentity);
 
-                SampleLogger.PrintSuccess($"Successfully created a new module identity with Id: '{moduleId}'");
+                SampleLogger.PrintSuccess($"Successfully created a new module identity with Id: '{moduleId}', ETag: '{response.Value.Etag}'");
```

Comment:
In all of these log statements I would recommend adding the device Id as well (since a module is always associated with a device, and its Id is unique only within a device).